service/src/main/java/be/portal/job/services/impls/ApplicationServiceImpl.java

```diff
@@ -100,7 +101,9 @@ public ApplicationResponse addApplication(ApplicationRequest applicationRequest)
         JobOffer jobOffer = jobOfferRepository.findById(applicationRequest.jobOfferId())
                 .orElseThrow(JobOfferNotFoundException::new);
 
-        Application application = applicationMapper.toEntity(jobSeeker, jobOffer, ApplicationStatus.SUBMITTED);
+        LocalDateTime applyDate = LocalDateTime.now();
```

Comment:
Since you are generating the date here, remove it from the ApplicationRequest because it is not needed anymore. Also make sure to check on the front-end to only send an object containing the jobOfferId